server/notification-providers/pushplus.js

```diff
@@ -0,0 +1,54 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { DOWN, UP } = require("../../src/util");
+
+class PushPlus extends NotificationProvider {
+    name = "PushPlus";
+
+    /**
+     * @inheritdoc
+     * @param {BeanModel} notification Notification object
+     * @param {string} msg Message content
+     * @param {?object} monitorJSON Monitor details
+     * @param {?object} heartbeatJSON Heartbeat details
+     * @returns {Promise<string>} Success message
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+        const url = "https://www.pushplus.plus/send";
+        try {
+            await axios.post(url, {
+                "token": notification.pushPlusSendKey,
+                "title": this.checkStatus(heartbeatJSON, monitorJSON),
+                "content": msg,
+                "template": "html"
+            }, { headers: { "Content-Type": "application/json" } }
+            );
+            return okMsg;
+
```

Comment:
nit: we usually format this like the following

```suggestion
            const config = {
                headers: {
                    "Content-Type": "application/json",
                },
            };
            const params = {
                "token": notification.pushPlusSendKey,
                "title": this.checkStatus(heartbeatJSON, monitorJSON),
                "content": msg,
                "template": "html"
            };
            await axios.post(url, params, config);
```